package/firmware/ipq-wifi/Makefile

```diff
@@ -1,6 +1,5 @@
 include $(TOPDIR)/rules.mk
 include $(INCLUDE_DIR)/version.mk
-
```

Comment:
Yes, indeed ... had to manually solve a merge conflict after rebasing because a few BFD have been upstreamed (commit 8217f02a1c5e) and it seems I deleted a line to many. I'll correct it.